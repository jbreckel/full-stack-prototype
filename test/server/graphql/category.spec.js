import gql from 'graphql-tag'

import execute from './mockServer'

describe('category', () => {
  it('can query categories', (done) => {
    execute(gql`
      query {
        categories {
          id
          name
        }
      }
    `)
      .then((result) => {
        expect(result).toMatchSnapshot()
        done()
      })
  })

  it('can query category with id', (done) => {
    execute(gql`
      query getCategory($id: ID!){
        category(id: $id) {
          id
          name
        }
      }
    `,
      {
        id: 'id',
      })
      .then((result) => {
        expect(result).toMatchSnapshot()
        done()
      })
  })
})
