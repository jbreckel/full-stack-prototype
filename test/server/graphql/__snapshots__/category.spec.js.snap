// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`category can query categories 1`] = `
Object {
  "data": Object {
    "categories": Array [
      Object {
        "id": "id",
        "name": "name",
      },
      Object {
        "id": "id",
        "name": "name",
      },
    ],
  },
}
`;

exports[`category can query category with id 1`] = `
Object {
  "data": Object {
    "category": Object {
      "id": "id",
      "name": "name",
    },
  },
}
`;
